common/djangoapps/track/views/segmentio.py

```diff
@@ -67,7 +65,7 @@ def segmentio_event(request):
     try:
         track_segmentio_event(request)
     except EventValidationError as err:
-        log.warning(
+        log.debug(
             'Unable to process event received from Segment: message="%s" event="%s"',
             str(err),
             request.body
```

Comment:
Per @mulby (which I agree), they are not warnings really - since they are expected cases.  However, we do want to still log them in case there's ever an issue that we want to debug.